lib/ui/playground/playground_button_present_alert.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:farmsmart_flutter/ui/common/Alert.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class PlaygroundButtonPresentAlertViewModel {
+  Function(Alert, BuildContext) listener;
+  Alert alert;
+
+  PlaygroundButtonPresentAlertViewModel({
```

Comment:
a bit overkill here to have a VM

Reply:
Yep for me it's an atom. It's also for the playground